test/extensions/quic_listeners/quiche/active_quic_listener_test.cc

```diff
@@ -270,5 +300,16 @@ TEST_P(ActiveQuicListenerTest, ProcessBufferedChlos) {
   ReadFromClientSockets();
 }
 
+TEST_P(ActiveQuicListenerTest, QuicProcessingDisabled) {
+  EnvoyQuicDispatcher* const envoy_quic_dispatcher =
+      ActiveQuicListenerPeer::quic_dispatcher(*quic_listener_);
+  EXPECT_CALL(runtime_.snapshot_, getBoolean("quic.enabled", true)).WillRepeatedly(Return(false));
+  SendFullCHLO(quic::test::TestConnectionId(1));
+  dispatcher_->run(Event::Dispatcher::RunType::NonBlock);
+  // If listener was enabled, there should have been session created for active connection.
+  EXPECT_TRUE(envoy_quic_dispatcher->session_map().empty());
+  ReadFromClientSockets();
```

Comment:
This is not necessary as the listener is black-holing all the packets.

Reply:
fixed